nixos/modules/services/web-apps/nextcloud.nix

```
@@ -360,6 +370,8 @@ in {
       };
 
       systemd.services = {
+        phpfpm-nextcloud.restartTriggers = [ cfg.package ];
```

What about providing a bit of context?

```suggestion
        # When upgrading the Nextcloud package, Nextcloud can report errors such as 
        # "The files of the app [all apps in /var/lib/nextcloud/apps] were not replaced correctly"
        # Restarting phpfpm on Nextcloud package update fix these issues (but this is a workaround).
        phpfpm-nextcloud.restartTriggers = [ cfg.package ];
```